packages/cli/src/commands/deploy/index.ts

```diff
@@ -453,6 +453,24 @@ export default async (client: Client) => {
   let deployStamp = stamp();
   let deployment = null;
 
+  // const localConfigurationOverrides = {}
+  // let overrideDetected = false;
+  // ['buildCommand', 'devCommand'].forEach(setting => {
+  //   const value = localConfig?.[setting]
+  //   if (value) {
+  //     overrideDetected = true;
+  //     localConfigurationOverrides[setting === 'ignoreCommand' ? 'commandForIgnoringBuildStep' : value]
+  //   }
+  // })
```

Comment:
Is this comment still needed or can we delete it?

Reply:
deleting